src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -565,28 +573,150 @@ ApiGlobe.prototype.setCenterAdvanced = function(pPosition /*, pDisableAnimationo
     this.setTilt(pPosition.tilt);
 };
 
+var updateTargetCamera = function(api){
+    api.scene.currentControls().updateCameraTransformation();
+    api.viewerDiv.dispatchEvent(eventRange);
+    api.removeEventListener('globe-loaded', updateTargetCamera);
+};
 /**
  * Sets the "range": the distance in meters between the camera and the current central point on the screen.
  * <iframe width="100%" height="400" src="//jsfiddle.net/iTownsIGN/Lt3jL5pd/embedded/" allowfullscreen="allowfullscreen" frameborder="0"></iframe>
  * @constructor
  * @param {Number} pRange - The camera altitude.
  * @param {Boolean} [pDisableAnimation] - Used to force the non use of animation if its enable.
  */
-
-ApiGlobe.prototype.setRange = function(pRange /*, bool*/ ) {
+ApiGlobe.prototype.setRange = function(pRange /*, bool anim*/) {
     eventRange.oldRange = this.getRange();
+    loaded = false;
```

Comment:
Yes, the event represents the tiles are loaded. In this case, I need to know when the tiles are loaded to compute the new target camera.